src/herbs2gql.js

```diff
@@ -7,4 +7,5 @@ module.exports = {
     defaultResolver: require('./defaultResolver'),
     defaultErrorHandler: require('./defaultErrorHandler').defaultErrorHandler,
     args2request: require('./args2request'),
+    herbarium2gql:require('./herbarium2gql')
```

Comment:
`herbarium2gql` ->  `herbs2gql` 

The idea is that Herbarium is not the important part here, it is just part of Herbs

Reply:
ok